components/backup-stream/src/endpoint.rs

```diff
@@ -857,20 +865,36 @@ where
         }
     }
 
-    pub fn on_force_flush(&self, task: String) {
+    pub fn on_force_flush(&self, task: TaskSelectorRef<'_>, sender: Sender<FlushResult>) {
```

Comment:
do we need to prevent too much force flush here if the callers abuse it?

Reply:
For now we don't need to care this. Because only 1 task is allowed.